### DISCUSSION TITLE
Is there an official Codesandbox or StackBlitz base for reproductions?

### DISCUSSION THREAD

**Discussion by dave-db** [mdx-editor/editor#319] [Q&A] [answered]
I'd like to create some proper repros for some small issues I've hit, to aid in fixing (and make sure it's not just my setup). Is there a base image/template I can fork?

**Comment by petyosi** [comment, marked as answer]
I usually start from here: https://codesandbox.io/p/sandbox/mdx-editor-base-q8s7zr?file=/src/App.tsx
